### PR TITLE
Sort tokens by ticker symbol in UI

### PR DESCRIPTION
**Pull request by webmaster128**

### CODE REVIEW

**Comment by apanizo** [comment]
Travis automatic deployment:
 https://pr327-327_wallet-demo_iov.surge.sh 
 
 Storybook book automatic deployment: 
 https://storybook_pr327-327_wallet-demo_iov.surge.sh

File: src/routes/balance/components/index.tsx [hunk 1]
```diff
@@ -106,6 +106,7 @@ class BalanceLayout extends React.Component<Props, State> {
     const info: Order = { xs: 2 };
     const grow: Order = { xs: 3 };
     const hasTokens = tokens && tokens.length;
+    const sortedTokens = [...tokens].sort((a, b) => a.tokenTicker.localeCompare(b.tokenTicker));
```
Comment (apanizo) [inline src/routes/balance/components/index.tsx]: This sorting should be done in the selector, for having memoized them. Here:
https://github.com/iov-one/wallet-demo/blob/master/src/routes/balance/container/selector.ts#L14

File: src/routes/receivePayment/receiveNonIov/components/index.tsx [hunk 2]
```diff
@@ -59,6 +59,8 @@ class ReceiveNonIov extends React.Component<Props, RecieveNonIOVState> {
     const { tickersList, classes } = this.props;
     const { howItWorksHook, phoneHook, ticker } = this.state;
 
+    const sortedTickers = [...tickersList].sort((a, b) => a.name.localeCompare(b.name));
```
Comment (apanizo) [inline src/routes/receivePayment/receiveNonIov/components/index.tsx]: Same here, go to `../container/selector.ts` and implement it there.

File: src/routes/sendPayment/components/FillPayment/SendCard.tsx [hunk 3]
```diff
@@ -80,6 +80,8 @@ class SendCard extends React.Component<Props, State> {
       onUpdateBalanceToSend,
     } = this.props;
 
+    const sortedTickers = [...tickersWithBalance].sort((a, b) => a.name.localeCompare(b.name));
```
Comment (apanizo) [inline src/routes/sendPayment/components/FillPayment/SendCard.tsx]: And same here, visit ```../container/selector.ts```

**Comment by webmaster128** [comment]
Ah great feeback @apanizo. I was looking for the source of the props but did not find it. All updated.

**Pull request merged**